datalad/distribution/tests/test_dataset.py

```diff
@@ -194,3 +195,24 @@ def test_subdatasets(path):
     assert_true(ds.is_installed())
     eq_(ds.get_subdatasets(), [])
     # TODO actual submodule checkout is still there
+
+
+@with_tempfile(mkdir=True)
+def test_require_dataset(path):
+    # in this folder by default
+    assert_equal(
+        require_dataset(None).path,
+        abspath(os.path.curdir))
```

Comment:
addressed in https://github.com/datalad/datalad/pull/743/files#diff-3a1382ff7de824e88b78f20b8fb6fa35L239